Documentation/ceph-common-issues.md

```diff
@@ -954,6 +955,81 @@ If you are using Kubernetes version is v1.15 or older, you will see an error lik
 >```
 You need to create the CRDs found in `cluster/examples/kubernetes/ceph/pre-k8s-1.16`. Note that these pre-1.16 `apiextensions.k8s.io/v1beta1` CRDs are deprecated in k8s v1.16 and will no longer be supported from k8s v1.22.
 
+## Pod Using Ceph Block Storage `is still being used` Error
+
+### Symptoms
+
+If you are using the CSI rbd plugin and you observe an error from a pod using
+Ceph for block storage that includes the error string `is still being used`.
+E.g.:
+```text
+MountVolume.MountDevice failed for volume "pvc-272a60b6-a51a-455a-b269-c794e9629b83" : rpc error: code = Internal desc = rbd image replicapool/csi-vol-e23bc868-8788-11eb-ab3c-1a1f46590305 is still being used
+```
+It likely indicates that the object is still mounted as an `rbd` device on a
+node other than where the pod is being scheduled.  One possible cause for this
+scenario is that the pod was successful provisioned with a mounted volume and
+then drained from the node is was originally schedule upon.
+
+### Solution
+
+The `rbd` device needs to be unmounted **from the node upon which it is mounted**.
+
+First, identify the `csi-rbdplugin` pod which is mounting the object.  As there
+should be only one of these pods scheduled per node, we do not need to
+determine the actual node name.
+
+The method below for achieving this was provided by @lipore in
```

Comment:
In the official docs I'd suggest we reference the issue where the contributor can be found. Mentioning specific contributors is better in PRs and issues.
```suggestion
The method below for achieving this was provided in [this issue](https://github.com/rook/rook/issues/4772#issuecomment-601064683)
```

Reply:
done